Fixes: Issue 114

YukiP7 opened the pull request:
Fixes : Issue #114 

I have recently developed individual pages specifically for the "About" and "Features" sections of the website. This allows users to navigate directly to each section with ease. I have taken great care to ensure that any modifications I implemented do not interfere with the overall website code.

Screenshots :
![Screenshot 2024-06-11 193635](https://github.com/chirantanbanik/Wellread/assets/109288720/48a9debc-ce17-4d4c-a656-902757ac7c4d)
![Screenshot 2024-06-11 193645](https://github.com/chirantanbanik/Wellread/assets/109288720/26393e95-a2e2-49c5-a18e-4b93fcdd5d37)


Please review the changes I have made and merge them accordingly.

chirantanbanik commented:
@YukiP7 There are merge conflicts.

![merge_conflicts](https://github.com/chirantanbanik/Wellread/assets/71392444/4d2a84db-748a-42b5-bf1e-656d8b8c5377)

You need to pull the changes and then make your changes, you have worked on the old code.

YukiP7 commented:
Hey, please check the code now. I have updated it. Also, there are changes
as I added two more pages: "about" and "features," and then directly
navigated it to the landing page. But I assure you there is no change in
the previous code; I only put it in another file.

On Tue, 11 Jun 2024 at 19:54, Chirantan Banik ***@***.***>
wrote:

> @YukiP7 <https://github.com/YukiP7> There are merge conflicts.
>
> merge_conflicts.jpeg (view on web)
> <https://github.com/chirantanbanik/Wellread/assets/71392444/4d2a84db-748a-42b5-bf1e-656d8b8c5377>
>
> You need to pull the changes and then make your changes, you have worked
> on the old code.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chirantanbanik/Wellread/pull/138#issuecomment-2160901796>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2BZ2EGKSR4SMB2VXCPZRYLZG4CCPAVCNFSM6AAAAABJEOARA6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNRQHEYDCNZZGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
